Getting-Started/Code/Debugging/Logging/index.md

```diff
@@ -91,27 +91,36 @@ If you are writing classes that inherit from one of these special Umbraco base c
 
 Then you can access the logging functionality via a special 'Logger' property included in those base classes and use the friendlier syntax of `Logger.Info<T>` to pass the type, if you add a reference to `Umbraco.Core.Logging` as a Using statement.
 
-Outside of these places, eg a ContentFinder or your own custom code, you can get a reference to the logger from the `Umbraco.Core.Composing` namespace, using the `Current.Logger` instance.
+Outside of these places, eg a ContentFinder or your own custom code, you can get a reference to the logger via Dependency Injection.
```

Comment:
```suggestion
Outside of these places, eg a ContentFinder or your own custom code, you can get a reference to the logger via Dependency Injection. While using Dependency Injection is the recommended way, it is possible to Current.Logger instead, if DI is not an option.

The following example uses DI.
```

Reply:
Hi Sophie, sorry I missed the notification of your comments. I'm happy with that. Thanks, t